.secrets.baseline

```diff
@@ -266,7 +266,7 @@
         "filename": "e2e/manifests/staging.toml",
         "hashed_secret": "f2400ac6800113efbeede5e2b7942549147600b3",
         "is_verified": false,
-        "line_number": 20
```

Comment:
I'd expect to see here the two new db passwords in the dev docker-compose (those that use the mock db).